site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +244,547 @@ They can do the same for Chrome on desktop by going to
+
+```http
+GET /accounts.php HTTP/1.1
+Host: idp.example
+Accept: application/json
+Cookie: 0x23223
+Sec-Fetch-Dest: webidentity
+```
+
+The browser expects a JSON response that includes an `accounts` property
+with an array of account information with following properties:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>id</code> (required)</td>
+    <td>Unique ID of the user.</td>
+  </tr>
+  <tr>
+    <td><code>name</code> (required)</td>
+    <td>Given and family name of the user.</td>
+  </tr>
+  <tr>
+    <td><code>email</code> (required)</td>
+    <td>Email address of the user.</td>
+  </tr>
+  <tr>
+    <td><code>given_name</code> (optional)</td>
+    <td>Given name of the user.</td>
+  </tr>
+  <tr>
+    <td><code>picture</code> (optional)</td>
+    <td>URL of the user avatar image.</td>
+  </tr>
+  <tr>
+    <td><code>approved_clients</code> (optional)</td>
+    <td>An array of RP client IDs which the user has registered with.</td>
+  </tr>
+</table>
+
+Example response body:
+
+```json
+{
+ "accounts": [{
+   "id": "1234",
+   "given_name": "John",
+   "name": "John Doe",
+   "email": "john_doe@idp.example",
+   "picture": "https://idp.example/profile/123",
+   "approved_clients": ["123", "456", "789"],
+  }, {
+   "id": "5678",
+   "given_name": "Johnny",
+   "name": "Johnny",
+   "email": "johnny@idp.example",
+   "picture": "https://idp.example/profile/456"
+   "approved_clients": ["abc", "def", "ghi"],
+  }]
+}
+```
+
+If the user is not signed in, respond with HTTP 401 (Unauthorized).
+
+The returned accounts list is consumed by the browser and will not be
+available to the RP.
+
+#### ID assertion endpoint {: #id-assertion-endpoint }
+
+The IdP's ID assertion endpoint returns an assertion for their signed-in user.
+When the user signs in to an RP website using [`navigator.credentials.get()`
+call](#sign-into-rp), the browser sends a `POST` request with cookies and a
+content type of `application/x-www-form-urlencoded` to this endpoint with the
+following information:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>client_id</code> (required)</td>
+    <td>The RP's client identifier.</td>
+  </tr>
+  <tr>
+     <td><code>account_id</code> (required)</td>
+     <td>The unique ID of the signing in user.</td>
+  </tr>
+  <tr>
+     <td><code>nonce</code> (optional)</td>
+     <td>The request nonce, provided by the RP.</td>
+  </tr>
+  <tr>
+     <td><code>disclosure_text_shown</code></td>
+     <td>Results in a string of <code>true</code> or <code>false</code> (rather than a boolean). The result is <code>false</code> if the RP's client ID was included in the <code>approved_clients</code> property list and the sign-in dialog displayed both the privacy policy and the terms of service.</td>
```

Comment:
if it's false, it means that disclosure text was not shown. i.e. the dialog displayed neither of them

Reply:
Fixed. Please check.